src/backend/src/entities/player.entity.ts

```diff
@@ -2,22 +2,28 @@ import {
   Entity,
   Column,
   PrimaryGeneratedColumn,
-  OneToOne,
-  JoinColumn,
   ManyToOne,
   BaseEntity,
 } from 'typeorm';
 import { Game } from './game.entity';
-import { User } from './user.entity';
+
+// IDK, just placeholder values for now
+export enum EmotionIdEnum {
+  Calm,
+  Greedy,
+  NotGreedy,
+}
 
 @Entity()
 export class Player extends BaseEntity {
-  @PrimaryGeneratedColumn()
+  @PrimaryGeneratedColumn() // Do we even need this anymore?
   id: number;
 
-  @OneToOne((type) => User)
-  @JoinColumn()
-  user: User;
+  @Column()
```

Comment:
I thought the plan here was to fully combine player and user. If thats the case then we only need the primary generated column right?

Reply:
Well a Player still has: an Id (given from qualtrics), the emotionId, and the Game that it's associated with right? Or are you saying we merge Player into User? I was originally just planning on making Player what we wanted and removing all User stuff in a separate PR.

Reply:
yeah im suggesting that we merge them. We can have playerId (ours), qualtricsId, and emotionId since that's all we care about, but we can discuss later